### PR TITLE
Toolbar: implement classes option

### PR DESCRIPTION
**Pull request by cgack**
Fixes gh-7687

New PR based off of 1.5-dev

### CODE REVIEW

File: css/structure/jquery.mobile.toolbar.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	position: absolute;
+	top: .24em;
+}
+.ui-toolbar-header-button-left {
+	left: .4em;
+}
+.ui-toolbar-header-button-right {
+	right: .4em;
+}
+.ui-btn-icon-notext.ui-toolbar-header-button-left {
```
Comment (arschmitz) [inline css/structure/jquery.mobile.toolbar.css]: Iets remove `.ui-btn-icon-notext` and use `.ui-toolbar ui-toolbar-header-button-left`

File: css/structure/jquery.mobile.toolbar.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	top: .3125em;
+	left: .3125em;
+}
+.ui-btn-icon-notext.ui-toolbar-header-button-right {
```
Comment (arschmitz) [inline css/structure/jquery.mobile.toolbar.css]: same here

File: js/widgets/fixedToolbar.js [hunk 3]
```diff
@@ -54,17 +55,23 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 				this._makeFixed();
 			}
 			if ( this.options.position === "fixed" && !this.options.supportBlacklist() ) {
-				var $page = ( !!this.page )? this.page: ( $(".ui-page-active").length > 0 )? $(".ui-page-active"): $(".ui-page").eq(0);
+				var pageActive = $( ".ui-page-active" );
```
Comment (arschmitz) [inline js/widgets/fixedToolbar.js]: lets rename this activePage to match the var below

File: js/widgets/fixedToolbar.js [hunk 4]
```diff
@@ -72,15 +79,17 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
...
 
-			if ( tclass && tclass !== "none" ) {
+			if ( transitionClass && transitionClass !== "none" ) {
 				// use appropriate slide for header or footer
-				if ( tclass === "slide" ) {
-					tclass = this.element.hasClass( "ui-header" ) ? "slidedown" : "slideup";
+				if ( transitionClass === "slide" ) {
+					transitionClass = this.role === "header" ?
+					"slidedown" :
+					"slideup";
```
Comment (arschmitz) [inline js/widgets/fixedToolbar.js]: this should all fit on one line now

**Comment by cgack** [comment]
@arschmitz more changes for your review. Thanks!

**Comment by cgack** [comment]
replacing with 8114

**Pull request closed without merging**